tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,453 @@
+
+See nyc_data_setup.sql [Github link: https://github.com/timescale/examples/tree/master/hello-timescale ] below which defines the schema for our 3 tables:
+
+> -- nyc_data_setup.sql
+> -- Create table 'rides' which will store trip data <br>
+> `DROP TABLE IF EXISTS "rides";` <br>
+> `CREATE TABLE "rides"( ` <br>
+>    `vendor_id TEXT,` <br>
+>    `pickup_datetime TIMESTAMP WITHOUT TIME ZONE NOT NULL,` <br>
+>    `dropoff_datetime TIMESTAMP WITHOUT TIME ZONE NOT NULL,` <br>
+>    `passenger_count NUMERIC,` <br>
+>    `trip_distance NUMERIC,` <br>
+>    `pickup_longitude  NUMERIC,` <br>
+>    `pickup_latitude   NUMERIC,` <br>
+>    `rate_code         INTEGER,` <br>
+>    `dropoff_longitude NUMERIC,` <br>
+>    `dropoff_latitude  NUMERIC,` <br>
+>    `payment_type INTEGER,` <br>
+>    `fare_amount NUMERIC,` <br>
+>    `extra NUMERIC,` <br>
+>    `mta_tax NUMERIC,` <br>
+>    `tip_amount NUMERIC,` <br>
+>    `tolls_amount NUMERIC,` <br>
+>    `improvement_surcharge NUMERIC,` <br>
+>    `total_amount NUMERIC` <br>
+> `);` <br>
+
+> -- Create hypertable for rides <br>
+> -- This allows us to take advantage of timescaledb's space and time partitioning <br>
+> `SELECT` `create_hypertable('rides', 'pickup_datetime', 'payment_type', 2, create_default_indexes=>FALSE)`; <br>
+> -- Create indexes (special look up tables/ pointers) on the following columns to speed up data retrieval <br>
+> `CREATE INDEX ON` rides (vendor_id, pickup_datetime desc); <br>
+> `CREATE INDEX ON` rides (pickup_datetime desc, vendor_id); <br>
+> `CREATE INDEX ON` rides (rate_code, pickup_datetime DESC); <br>
+> `CREATE INDEX ON` rides (passenger_count, pickup_datetime desc); <br>
+>
+> -- Create table 'payment_types' to store description of payment types for easy lookup <br>
+> `CREATE TABLE IF NOT EXISTS` "payment_types"( <br>
+>    payment_type INTEGER, <br>
+>    description TEXT <br>
+> ); <br>
+> `INSERT INTO` payment_types(payment_type, description) `VALUES` <br>
+> (1, 'credit card'), <br>
+> (2, 'cash'), <br>
+> (3, 'no charge'), <br>
+> (4, 'dispute'), <br>
+> (5, 'unknown'), <br>
+> (6, 'voided trip'); <br>
+
+> -- Create table 'rates' to store description of rate codes for for easy lookup <br>
+> `CREATE TABLE IF NOT EXISTS` "rates"( <br>
+>    rate_code   INTEGER, <br>
+>    description TEXT <br>
+> ); <br>
+> `INSERT INTO` rates(rate_code, description) `VALUES` <br>
+> (1, 'standard rate'), <br>
+> (2, 'JFK'), <br>
+> (3, 'Newark'), <br>
+> (4, 'Nassau or Westchester'), <br>
+> (5, 'negotiated fare'), <br>
+> (6, 'group ride'); <br>
+
+### 1.3 Load trip data into timescaledb
+
+[Note, if the dataset is already pre-loaded into your Timescale Cloud instance, you can skip straight to Mission 2.]
+
+Next, let’s upload the taxi cab data into your Timescale Cloud instance. The data is in a file called nyc_data_rides.csv Download it here [Github Link: https://github.com/timescale/examples/tree/master/hello-timescale].
```

Comment:
Make the text 'nyc_data_rides.csv' into the link, make the link directly to the file and add a `DOWNLOAD_LINK` tag to create the download link icon (see https://github.com/timescale/docs.timescale.com-content/blob/master/tutorials/other-sample-datasets.md for an example of adding the `DOWNLOAD_LINK` tag)